### PR TITLE
some wrong max/min layer height corrected and 0.15 nozzle added for Voron 2.4

### PR DESCRIPTION
**Pull request by lukicsl**
There were some errors at the last pull request where the min layer height was higher than the defined profiles and the max was in some places way too high.
Added 0.15 nozzle, because I have one :)

### CODE REVIEW

File: resources/profiles/Voron/machine/Voron 2.4 250 0.6 nozzle.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
       "0.15"
     ],
       "nozzle_diameter": [
-      "0.6"
```
Comment (SoftFever) [inline resources/profiles/Voron/machine/Voron 2.4 250 0.6 nozzle.json:16]: is this a typo?

Comment (lukicsl) [reply resources/profiles/Voron/machine/Voron 2.4 250 0.6 nozzle.json:16]: no,
"layer_height": "0.0",

**Review by SoftFever — APPROVED** [review]
Looks good.

**Comment by SoftFever** [comment]
merged.
Thank you 👍

**Pull request merged**